index.bs

```diff
@@ -690,6 +692,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |issuedRequests| be a new [=ordered set=]. 
 
+1. If no |authenticators| is available on this platform, return a {{DOMException}} whose name is "{{NotFoundError}}", 
```

Comment:
s/If no |authenticators| is available/If there are no [=authenticators=] currently available/